onboarding/onboarding.md

```diff
@@ -76,3 +76,16 @@ you should reopen your terminal to find "(base)" next to your prompt: ![](conda_
     ```
 
 1. [Add a GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-new-gpg-key-to-your-github-account ) to your Modin account. Your commits need to be signed with a GPG key. For mac, you can use [Mac GPG](https://gpgtools.org/).
+
+
+1. (Optional) We recommend a few workflow settings:
+
+    1. If you use Visual Studio Code, auto-format with black every time you save changes:
+        1. Install [Microsoft's Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+        1. Open your VSCode settings, in `Code -> Preferences -> Settings`.
+        1. Search for "python formatting provider" and select "black" from the dropdown menu.
```

Comment:
nit: link to black docs or repo for those unfamiliar with this formatter

Reply:
Done.